.github/workflows/go.yml

```diff
@@ -54,4 +60,63 @@ jobs:
       run: |
         cd $GOPATH/src/github.com/${{ github.repository }}
         ./test/install_bitcoind.sh
-        ./test/csn_bridge.sh "./cmd/utreexoserver/utreexoserver" "./cmd/utreexoclient/utreexoclient"
+        ./test/csn_bridge.sh "./cmd/utreexoserver/utreexoserver" "./cmd/utreexoclient/utreexoclient" "${{ github.workspace }}/testdatapath"
+
+    - name: Upload testdata
+      uses: actions/upload-artifact@v2
+      with:
+        name: testdatapath
+        path: "${{ github.workspace }}/testdatapath"
+
+  check-proof-backwards:
+    name: Check proof backwards compatibility
+    runs-on: ubuntu-latest
+    needs: Build
+    timeout-minutes: 5
+    steps:
+    # set GOPATH
+    - name: setup env
+      shell: bash
+      run: |
+        echo "${{ github.workspace }}/go/bin:" >> $GITHUB_PATH
+        echo "GOPATH=${{ github.workspace }}/go" >> $GITHUB_ENV
+
+    - uses: actions/download-artifact@v2
+      with:
+        name: testdatapath
+
+    - name: Install Go
+      if: success()
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        ref: master
+        path: go/src/github.com/${{ github.repository }}
+
+    - name: Get dependencies
+      run: |
+        cd $GOPATH/src/github.com/${{ github.repository }}
+        go get -v -d ./...
+
+    - name: Build
+      shell: bash
+      run: |
+        cd $GOPATH/src/github.com/${{ github.repository }}/cmd/utreexoserver
+        go install -v
+
+    - name: checkout
```

Comment:
Is this only needed because `./test/check_proofs_backwards.sh` does not yet exist on master?

Reply:
Yes. Not exactly the prettiest... I guess we could update this afterwards